src/main/java/com/garbagemule/MobArena/commands/setup/SetupCommand.java

```diff
@@ -350,8 +351,9 @@ private boolean warps(PlayerInteractEvent event) {
                     }
                     next = formatYellow("Current warp: %s", warpArray[warpIndex]);
                     return true;
-            }
-            return false;
+                default:
+                    return false;
+                }
```

Comment:
There's something weird about the curlies here. I think line 356 just needs to be "dedented" by one indent level because it belongs to the `switch` statement and not the `default` branch.

Reply:
Fixed in eb8cfac07f20f3b92b331fdc74f03c0a7d6565aa